graylog2-server/src/main/java/org/graylog2/contentpacks/facades/EntityFacade.java

```diff
@@ -20,6 +20,8 @@
 import com.google.common.graph.GraphBuilder;
 import com.google.common.graph.ImmutableGraph;
 import com.google.common.graph.MutableGraph;
+import org.graylog2.contentpacks.model.constraints.Constraint;
+import org.graylog2.contentpacks.model.constraints.GraylogVersionConstraint;
```

Comment:
These are unused imports. Please run "Optimize Imports" on the complete "contentpacks" package in IntelliJ to remove all of them. :smiley: